libtcmu_common.h

```diff
@@ -109,6 +109,8 @@ void tcmu_dev_set_private(struct tcmu_device *dev, void *priv);
 void *tcmu_get_daemon_dev_private(struct tcmu_device *dev);
 void tcmu_set_daemon_dev_private(struct tcmu_device *dev, void *priv);
 int tcmu_dev_get_fd(struct tcmu_device *dev);
+const char *tcmu_dev_get_memory_info(struct tcmu_device *dev,
+	void **base, size_t *len, off_t *offset);
```

Comment:
Follow the formatting in the file where this line should be tabbed and possibly spaced over more to match up with the char after the "(".